xml/System.ComponentModel.Design/ObjectSelectorEditor.xml

```diff
@@ -114,9 +114,8 @@
         <Parameter Name="treeView" Type="System.Windows.Forms.TreeView" Index="0" FrameworkAlternate="netcore-3.0;netcore-3.1" />
       </Parameters>
       <Docs>
-        <param name="treeView">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="treeView">The tree view control to modify.</param>
+        <summary>Modify a WinForms TreeView control to use the new Explorer style theme.</summary>
```

Comment:
Do we use "_WinForms_" term throughout the docs?

Reply:
@dotnet/docs what is the preferred term to refer to WinForms in the docs?

Reply:
ping @Thraka

Reply:
For Docs I usually think I see Windows Forms. But, in code I wouldn't be surprised to see WinForms at all.

Reply:
```suggestion
        <summary>Modify a Windows Forms TreeView control to use the new Explorer style theme.</summary>
```